contracts/v0.8/VerifRegAPI.sol

```diff
@@ -84,52 +63,27 @@ library VerifRegAPI {
     function extendClaimTerms(VerifRegTypes.ExtendClaimTermsParams memory params) internal returns (CommonTypes.BatchReturn memory) {
         bytes memory raw_request = params.serializeExtendClaimTermsParams();
 
-        bytes memory raw_response = Actor.call(
-            VerifRegTypes.ExtendClaimTermsMethodNum,
-            VerifRegTypes.ActorID,
-            raw_request,
-            Misc.CBOR_CODEC,
-            msg.value,
-            false
-        );
+        bytes memory raw_response = Actor.call(VerifRegTypes.ExtendClaimTermsMethodNum, VerifRegTypes.ActorID, raw_request, Misc.CBOR_CODEC, 0, false);
 
         bytes memory result = Actor.readRespData(raw_response);
 
         return result.deserializeBatchReturn();
     }
 
-    function removeExpiredClaims(
-        VerifRegTypes.RemoveExpiredClaimsParams memory params
-    ) internal returns (VerifRegTypes.RemoveExpiredClaimsReturn memory) {
+    function removeExpiredClaims(VerifRegTypes.RemoveExpiredClaimsParams memory params) internal returns (VerifRegTypes.RemoveExpiredClaimsReturn memory) {
         bytes memory raw_request = params.serializeRemoveExpiredClaimsParams();
 
-        bytes memory raw_response = Actor.call(
-            VerifRegTypes.RemoveExpiredClaimsMethodNum,
-            VerifRegTypes.ActorID,
-            raw_request,
-            Misc.CBOR_CODEC,
-            msg.value,
-            false
-        );
+        bytes memory raw_response = Actor.call(VerifRegTypes.RemoveExpiredClaimsMethodNum, VerifRegTypes.ActorID, raw_request, Misc.CBOR_CODEC, 0, false);
 
         bytes memory result = Actor.readRespData(raw_response);
 
         return result.deserializeRemoveExpiredClaimsReturn();
     }
 
-    function universalReceiverHook(
-        VerifRegTypes.UniversalReceiverParams memory params
-    ) internal returns (VerifRegTypes.AllocationsResponse memory) {
+    function universalReceiverHook(VerifRegTypes.UniversalReceiverParams memory params) internal returns (VerifRegTypes.AllocationsResponse memory) {
         bytes memory raw_request = params.serializeUniversalReceiverParams();
 
-        bytes memory raw_response = Actor.call(
-            VerifRegTypes.UniversalReceiverMethodNum,
-            VerifRegTypes.ActorID,
-            raw_request,
-            Misc.CBOR_CODEC,
-            msg.value,
-            false
-        );
+        bytes memory raw_response = Actor.call(VerifRegTypes.UniversalReceiverMethodNum, VerifRegTypes.ActorID, raw_request, Misc.CBOR_CODEC, 0, false);
 
         bytes memory result = Actor.readRespData(raw_response);
```

Comment:
I don't think we need this method here at all. AFAIK, this is used by some standards to validate that the contract can receive tokens, but it does not need to be exposed to the user _on a specific actor_. If anything, I would create a helper function that allows the dev to call the universal receiver hook on _any_ address they choose.

Reply:
The function was marked to be exported as far as I remember. We can remove it in another PR if necessary though.